mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/logging/impl/Log.java

```diff
@@ -788,4 +788,23 @@ ConstructorProjectionApplicationException errorApplyingProjectionConstructor(
 
 	@Message(id = ID_OFFSET + 127, value = "Unable to export the schema: %1$s" )
 	SearchException unableToExportSchema(String cause, @Cause Exception e, @Param EventContext context);
+
+	@Message(id = ID_OFFSET + 128,
+			value = "Indexing plan for '%1$s' cannot be created as this type is excluded by the indexing plan filter.")
+	SearchException attemptToCreateIndexingPlanForExcludedType(PojoRawTypeIdentifier<?> typeIdentifier);
+
+	@Message(id = ID_OFFSET + 129,
+			value = "'%1$s' cannot be included and excluded at the same time within one filter. " +
+					"Already included types: '%2$s'. " +
+					"Already excluded types: '%3$s'.")
+	SearchException indexingPlanFilterCannotIncludeExcludeSameType(PojoRawTypeIdentifier<?> typeIdentifier,
+			Set<PojoRawTypeIdentifier<?>> includes, Set<PojoRawTypeIdentifier<?>> excludes);
+
+	@Message(id = ID_OFFSET + 130,
+			value = "No matching entity type for class '%1$s'."
+					+ " This class is neither an entity type mapped in Hibernate Search nor a superclass of such entity type."
+					+ " Note interfaces are not considered superclasses and are not permitted here."
+					+ " Valid classes are: %2$s")
+	SearchException unknownClassForNonInterfaceSuperType(Class<?> invalidClass,
+			Collection<Class<?>> validClasses);
```

Comment:
Just noticed this wasn't using formatters. You might want to use:

* `@FormatWith(ClassFormatter.class)`
* `@FormatWith(CommaSeparatedClassesFormatter.class)`